tests/dummy/app/pods/details/template.hbs

```diff
@@ -8,8 +8,8 @@
   {{/block-slot}}
 
   {{#block-slot slot 'related-route' as |routes|}}
-    {{#routes.related 'details.related.devices' icon='frost/network-element'}}Devices{{/routes.related}}
-    {{#routes.related 'details.related.friends' 3 icon='frost/tenant'}}Friends{{/routes.related}}
-    {{#routes.related 'details.related.subscriptions' icon='frost/service'}}Subs{{/routes.related}}
+    {{#routes.related 'details.related.devices' icon='network-element'}}Devices{{/routes.related}}
```

Comment:
not sure if devices are related to friends and subscriptions. Can we make all of them generic and non networking related? so replace devices with something else?